src/Containers.jsx

```diff
@@ -837,9 +891,9 @@
                                         else if (b == "no-pod") return 1;
 
                                         // User pods are in front of system ones
-                                        if (this.props.pods[a].uid !== this.props.pods[b].uid)
-                                            return this.props.pods[a].uid === 0 ? 1 : -1;
-                                        return this.props.pods[a].Name > this.props.pods[b].Name ? 1 : -1;
+                                        if (pods[a].uid !== pods[b].uid)
+                                            return pods[a].uid === 0 ? 1 : -1;
+                                        return pods[a].Name > pods[b].Name ? 1 : -1;
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2266-29c256e1-20250917-140235-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)